.eslintignore

```diff
@@ -1 +1,2 @@
-build
\ No newline at end of file
+dist
+test
```

Comment:
I think the tests should be linted too, no? If there's something broken in them then they may not be testing what they are supposed to?

Reply:
Ah, I presume this is to fix CI. I think, especially if you want to use this as a template for other TS fastify plugins, we should figure out a way to have the tests included in the TS project context so they can be typecheck'd and linted. If we have automated tooling that doesn't cost human time that helps make code better, why should we exclude any code from it?

I think this means we may have to go to the two tsconfig solution which kinda stinks. I haven't figured out yet why exactly but this one file setup seems to work for `tsdx` which is used in a lot of projects, and seems like a good golden path to try to replicate.

Reply:
eslint/typescript is complaining that that folder isn't defined in tsconfig.json in the include path. Maybe we can add it somehow? (without having the double tsconfig.json). Any clues?

Reply:
Fixed with a custom `tsconfig.eslint.json` as suggested on eslint/typescript docs.